app/Http/Resources/GroupResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class GroupResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'time_created' => $this->created_at->toDateTimeString(),
+            'time_updated' => $this->updated_at->toDateTimeString(),
+            'name' => $this->name,
+            'description' => $this->description,
+            'is_public' => $this->is_public,
```

Comment:
- [ ] This will return a 0 or a 1, it's preferred to return a boolean by writing `!!$this->is_public`

Reply:
did `(boolean) $this->is_public`